### PR TITLE
Iter var to path

### PR DESCRIPTION
**Pull request by MirijaHa**
As suggested by @daisy-pacheco we include the name of the iteration var in the output subdirs and stated that results will not be overwritten.
FYI @jakubcervenka

### CODE REVIEW

File: vignettes/articles/03-run-stress-test.Rmd [hunk 1]
```diff
@@ -126,17 +126,19 @@ run_stress_test(
 
 ### Output files
 The output files are written into a subdirectory of the path indicated as
-`output_path` in the function call. The name of the subdirectory is the
-timestamp of the analysis. It will be generated automatically.
+`output_path` in the function call. The name of the subdirectory is created from
+the timestamp of the analysis and a suffix. The suffix is `_standard` if
+stresstest is run with a fixed set of parameters. If a sensitivity analysis (see
+below) is run the name of varied the detail variable is used. The subdirectory
```
Comment (jacobvjk) [inline vignettes/articles/03-run-stress-test.Rmd:132]: order of words is off

File: vignettes/articles/03-run-stress-test.Rmd [hunk 2]
```diff
@@ -126,17 +126,19 @@ run_stress_test(
 
 ### Output files
 The output files are written into a subdirectory of the path indicated as
-`output_path` in the function call. The name of the subdirectory is the
-timestamp of the analysis. It will be generated automatically.
+`output_path` in the function call. The name of the subdirectory is created from
+the timestamp of the analysis and a suffix. The suffix is `_standard` if
+stresstest is run with a fixed set of parameters. If a sensitivity analysis (see
```
Comment (jacobvjk) [inline vignettes/articles/03-run-stress-test.Rmd:131]: while I may have used a similar formulation somewhere, I feel like "fixed set of parameters" is not intuitively clear to mean: one parameter per argument. "The suffix `_standard` is used when each argument uses a single parameter value."?

**Review by jacobvjk — COMMENTED** [review]
see comments

File: vignettes/articles/03-run-stress-test.Rmd (hunk 1 above)
Comment (MirijaHa) [reply vignettes/articles/03-run-stress-test.Rmd]: done

File: vignettes/articles/03-run-stress-test.Rmd (hunk 2 above)
Comment (MirijaHa) [reply vignettes/articles/03-run-stress-test.Rmd]: done

**Review by jacobvjk — APPROVED** [review]
lgtm

**Pull request merged**